lib/result-view.js

```diff
@@ -1,13 +1,36 @@
 "use babel";
 
 import { CompositeDisposable } from "atom";
-import { richestMimetype, transforms, displayOrder } from "@nteract/transforms";
+import { allowUnsafeNewFunction } from "loophole";
+import {
+  standardTransforms,
+  standardDisplayOrder,
+  registerTransform,
+  richestMimetype
+} from "@nteract/transforms";
 import React from "react";
 import ReactDOM from "react-dom";
 import { Map as ImmutableMap } from "immutable";
 
 import { log } from "./utils";
 
+// Due to Atom's CSP we need to use loophole :(
+let PlotlyTransform;
+allowUnsafeNewFunction(() => {
```

Comment:
Without `loophole` this is the error:

```
At Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".


EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

    at a (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:38:18417)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:38:18767
    at Object.n.441.cwise-compiler (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:38:19169)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:606
    at Object.n.138.ndarray (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:18:19768)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:606
    at Object.n.153../lib/shaders (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:20:1025)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:606
    at Object.n.950.../../constants/gl3d_dashes (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:71:7923)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:606
    at Object.n.952.../../constants/gl_markers (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:71:13413)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:606
    at Object.n.23.../src/traces/scatter3d (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:14751)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:606
    at Object.n.17../bar (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:13508)
    at i (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:555)
    at t (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:722)
    at /Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:739
    at l (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:85)
    at Object.<anonymous> (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/plotly/plotly.min.js:7:278)
    at Module._compile (/app.asar/src/native-compile-cache.js:109:30)
    at /app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/lukasgeiger/code/hydrogen/node_modules/@nteract/transform-plotly/lib/index.js:26:14)
    at Module._compile (/app.asar/src/native-compile-cache.js:109:30)
    at /app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/lukasgeiger/code/hydrogen/lib/result-view.js:20:19)
    at Module._compile (/app.asar/src/native-compile-cache.js:109:30)
    at /app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/lukasgeiger/code/hydrogen/lib/main.js:7:26)
    at Module._compile (/app.asar/src/native-compile-cache.js:109:30)
    at /app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/app.asar/src/package.js:796:27)
    at Package.module.exports.Package.activateNow (/app.asar/src/package.js:207:16)
    at /app.asar/src/package.js:876:25
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:240:20)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```